Samantha-samanthalpark/Homework-3/object-homework/app.js

```diff
@@ -0,0 +1,94 @@
+/////////////////////////////////////////////////////////////
+
+///////////////////////// Challenge /////////////////////////
+
+/////////////////////////////////////////////////////////////
+
+// Write your answers inside this file, at the places where it's indicated by the comments.
+
+
+// 1. Suppose that we wanted to create a browser-based calendar program for keeping track of things. In comments, list at least three reasonable abstractions that you might use to build this program; for each abstraction, list out at least two properties and behaviors that it would make sense for that abstraction to have. Below is an example - please write your answer in the same format.
+  // 'Car'
+  // Description: Our app manages car sales.
+  // Every 'car' has
+  // - a make
+  // - a model
+  // - a year
+  // Every 'car' can
+  // - drive
+  // - brake
+  // - park
+
+// Answer Starts Here
+
+// 'Calendar'
+// Description: Manage/keep track of events
+// Every 'event' has
+// - a day
+// - a month
+// - a year
+// - day of the week
+// - name
+// - location
+// - duration
+// Every 'calendar' can
+// - add event
+// - edit event
+// - move event
+// - delete event
+
+// Answer Ends Here
+
+
+// 2. Create an Object literal that lines up with the following description. Store it in the variable 'pet_owner', below. Be sure to give it reasonable values for each of its properties.
+
+  // 'Owner'
+  // Description: We are making an app for a veterinary clinic - it manages pets' vaccinations.
+  // Every 'owner' has:
+  // - a name
+  // - an address
+
+
+
+// Answer Starts Here
+let pet_owner = {
+  name = "first name last name",
+  address = "number street city state zipcode",
+  getOwnerInfo: function(){
+    console.log(this.name + ": " + this.address);
+  }
+}
+// Answer Ends Here
+
+
+// 3. Create an Object literal that lines up with the following description. Store it in the variable `some_pet`, below.
+
+  // Pet
+  // Description: We are making an app for a veterinary clinic - it manages pets' vaccinations.
+  // Every 'pet' has:
+  // - a name
+  // - a species
+  // - a breed
+  // - a noise that it can make (e.g. 'bark', 'meow', etc.)
+  // Every pet can:
+  // - make noise (each pet makes its own unique noise, as specified by `noise`.
+
+
+
+// Answer Starts Here
+let some_pet = {
+  name = "Bailey",
+  species = "Cat",
+  breed = "Bengal Shorthair Mix",
+  noiseMakes = "Meow",
```

Comment:
You have a property named `noiseMakes` and a method called `makeNoise` which could be a little confusing. In general, you want the names to be clear and descriptive and easily differentiated. So keep that in mind as you're naming things in the future -- it can take some work!